.eslintrc.js

```diff
@@ -15,9 +15,13 @@ module.exports = {
   ignorePatterns: ['.eslintrc.js'],
   rules: {
     '@typescript-eslint/interface-name-prefix': 'off',
-    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'error',
```

Comment:
I've set this to `error` as it is now our source of truth having removed assertions.